docs/writing-docs/mdx.md

```diff
@@ -254,36 +285,67 @@ import Changelog from '../CHANGELOG.md';
 
 -->
 
-## Linking to other stories and pages
+## Troubleshooting
 
-When writing MDX, you may want to provide links to other stories or documentation pages and sections. You can use the `path` query string.
+### The migration seems flaky and keeps failing
 
-Considering a story with ID `some--id`, this redirects to the **Docs** tab of the story:
+By default, running the [migration command](#automigration) will try and migrate all existing MDX files in your project according to the MDX 2 specification. However, this might not always be possible, and you might run into issues during the migration. To help you troubleshoot those issues, we've prepared some recommendations that might help you.
```

Comment:
I don't believe our migration tries to convert MDX1->2, or do I have that wrong @JReinhold @yannbf ?

Reply:
PS should this content be in our migration guide rather than the docs themselves? It doesn't see relevant to new users.